make/utils_exposed.cmake

```diff
@@ -290,7 +290,7 @@ function(DynamoRIO_force_static_link target lib)
     else ()
       set(incname "_dr_client_main")
     endif ()
-    append_property_string(TARGET ${target} LINK_FLAGS "/include:${incname}")
```

Comment:
Curiosity: How did this work before? (Or it didn't?) `append_property_string` is not defined here.

Reply:
I believe that external-project use did not work.  Internal use ends up using internally-defined versions so we didn't notice as we only have a few external-project tests; and I guess no external users were trying to static-link drmemtrace on Windows.